JOSS submission

charleygros opened the pull request:
The purpose of this PR is to:
- generate a paper.md and paper.bib for submission to JOSS (see [guidelines](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain)).
- move some of the text from the JMLR submission that cannot fit in the JOSS submission in the RTD

charleygros committed:
paper.md: skeleton

charleygros committed:
paper.bib

charleygros committed:
paper.md: title n authors

charleygros committed:
paper.md: statement of needs

charleygros committed:
paper.md: skeleton

charleygros committed:
paper.md: ackn

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,105 @@
+
+# Acknowledgements
+
+The authors thank Alexandru Jora, Nick Guenther, Christian Perone, Valentine Louis-Lucas, Benoît Sauty-De-Chalon, Alexandru Foias, Marie-Hélène Bourget and Leander Van Eekelen for their useful contributions, and Guillaume Dumas for proof-reading the manuscript. Funded by the Institut de valorisation des données (IVADO), the Canada Research Chair in Quantitative Magnetic Resonance Imaging [950-230 ...
```
> C.G has a fellowship from IVADOMED

One day `ivadomed` will be able to provide fellowships 😉

jcohenadad committed:
Reformatted ivadomed, fixed fellowship

jcohenadad committed:
Fixed MD formatting

jcohenadad committed:
Minor reformatting

charleygros committed:
Merge branch 'master' of https://github.com/neuropoly/ivado-medical-imaging into all/joss_submission

charleygros committed:
paper.md: rm aas params

charleygros committed:
images/overview.png: no title

charleygros committed:
paper.md: add loader

charleygros committed:
paper.md: add training

charleygros committed:
paper.md: add evaluation

charleygros commented:
The core of the text is now ([9c37f68](https://github.com/ivadomed/ivadomed/pull/502/commits/9c37f680d67da8606a4d21984e6c6482656922b4)) about 1300+ words, whereas:
```
The paper should be between 250-1000 words
```

charleygros commented:
Note from the guidelines:
```
Mention (if applicable) a representative set of past or ongoing research projects using the software and recent scholarly publications enabled by it.
```

What about: adding a sentence in the manuscript referring to a section in RTD where we document some of `ivadomed` user cases (eg the 3 user cases from the JMLR manuscript) and research papers (eg SoftSeg)?

charleygros committed:
docs/source/user_cases.rst: init

charleygros committed:
use_cases.rst: format

charleygros committed:
use_cases.rst: rename file

charleygros committed:
use_cases.rst: sct

charleygros committed:
use_cases.rst: exvivo

charleygros committed:
use_cases.rst: tumor

charleygros committed:
lemay_2020.png

charleygros committed:
use_cases.rst: REF

charleygros committed:
paper.md: usage

charleygros committed:
Merge branch 'master' of https://github.com/neuropoly/ivado-medical-imaging into all/joss_submission

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+Another challenge of medical imaging is the heterogeneity of the data across clinical centers, in terms of image features (e.g., contrast, resolution) and population demographics. This makes it challenging to create models that can generalize well across the many existing datasets. Recent cutting-edge methods address this problem, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], howeve ...
+
+# Software description
+
+`ivadomed` is based on PyTorch framework [@paszke2017automatic] with GPU acceleration supported by CUDA. It can easily be installed via [PyPI](https://pypi.org/project/ivadomed/) and the whole package is tested with a [continuous integration](https://github.com/ivadomed/ivadomed/actions?query=workflow\%3A\%22Run+tests\%22) framework. The project website, which includes user and API documentation, ...
+
```
I would remove because: 
- this section is optional
- the info is "somewhat" generic and can easily be found on the website
- we need to space

```suggestion
```

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+# Summary
+
+`ivadomed` is an open-source Python package for designing, end-to-end training, and evaluating deep learning models applied to medical imaging data. The package includes APIs, command-line tools, documentation, and tutorials. `ivadomed` also includes pre-trained models such as spinal tumor segmentation and vertebral labeling. Original features of `ivadomed` include a data loader that can parse im ...
```
Removing some meat already present in subsequent sections
```suggestion
`ivadomed` is an open-source Python package for designing, end-to-end training, and evaluating deep learning models applied to medical imaging data. The package includes APIs, command-line tools, documentation, and tutorials. `ivadomed` also includes pre-trained models such as spinal tumor segmentation and vertebral labeling. Original features of `ivadomed` include a data loader that can parse image and subject metadata for custom data splitting or extra information during training and evaluation. Any dataset following the [Brain Imaging Data Structure (BIDS)](https://bids.neuroimaging.io/) convention will be compatible with `ivadomed` without the need to manually organize the data. Beyond the traditional deep learning methods, `ivadomed` features cutting-edge architectures, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], as well as various uncertainty estimation methods (aleatoric and epistemic), and losses adapted to imbalanced classes and non-binary predictions. Example applications of `ivadomed` include MRI object detection, segmentation, and labeling of anatomical and pathological structures. `ivadomed`'s main project page is available at https://ivadomed.org.
```

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+# Statement of need
+
+Deep learning is increasingly used in medical image processing [@kim_deep_2019]. It provides automated solutions to repetitive and/or tedious tasks such as the segmentation of pathological structures. However, medical imaging data present many challenges: datasets are often not publicly-available, ground truth labels are scarce due to the limited availability of expert raters, and needs can be ve ...
```
```suggestion
Deep learning is increasingly used in medical image processing [@kim_deep_2019]. It provides automated solutions to repetitive and/or tedious tasks such as the segmentation of pathological structures. However, medical imaging data present many challenges: datasets are often not publicly-available, ground truth labels are scarce due to the limited availability of expert raters, and needs can be very specific and tailored to particular datasets and tasks. There already exists a variety of deep learning framework for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison-with-other-projects). 
```

charleygros replied:
I am a bit hesitant about integrating this one: we would be highlighting that (unfortunately) we are not the only one, without developing our strengths.

jcohenadad replied:
I think we need to mention it. At this point, there are already >15 similar toolboxes and ignoring them would be dishonest and would give the impression that we developed this toolbox in our bubble. If readers get that impression, this they will judge ivadomed very negatively. If we demonstrate we are aware of the state-of-the-art, it will give the impression that we can "only do better", because we know what's out there.  I'll find a rephrasing to spin ivadomed's contribution more positively.

charleygros replied:
Mmmm I agree! THanks for explaining.
I agree even more when reading the [JOSS checklist](https://joss.readthedocs.io/en/latest/review_checklist.html#software-paper):
```
State of the field: Do the authors describe how this software compares to other commonly-used packages?
```

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+## Loader
+
+An important aspect of machine learning is data management. Lots of time is usually spent manually organizing data into a proper structure to make the dataset compatible with a chosen analysis pipeline [@Willemink2020-au]. `ivadomed` features a data loader module that expects datasets to be structured according to a widely-accepted convention: the Brain Imaging Data Structure (BIDS) [@bids_2016]. ...
```
removed some meat
```suggestion
In machine learning, lots of time is spent curating the data (renaming, filtering per features) before entering the training pipeline [@Willemink2020-au]. `ivadomed` features an advanced data loader compatible with a standardized data structure in neuroimaging: the Brain Imaging Data Structure (BIDS) [@bids_2016]. Thus, any dataset following the BIDS convention can readily be used by `ivadomed`. BIDS convention is originally designed around MRI data and accepts NIfTI file formats, but the BIDS community is actively expanding its specifications to other modalities (CT, MEG/EEG, microscopy), which `ivadomed` will then be able to accommodate. 
```

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+# Usage
+
+Past or ongoing research projects using `ivadomed` are listed [here](https://github.com/ivadomed/ivadomed/docs/source/use_cases.rst).
```
i think it would be *really* cool to add Andréanne's figure about tumor segmentation. People *love* figures.

charleygros replied:
Yeah!! Done in [608a24d](https://github.com/ivadomed/ivadomed/pull/502/commits/608a24d9044de12cab84eb12acb37be4dacdcb5d).

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+## Training
+
+`ivadomed` includes all the necessary components for training segmentation models from start to finish. The first step usually consists of applying data augmentation transformations, such as affine, elastic, and ground-truth dilation, which are all included in `ivadomed`. The next step is model training, which can be done either from scratch or using transfer learning on a pre-trained model by fr ...
```
Removed a bit of meat, added ref to Figure 2 (tumor seg)
```suggestion
`ivadomed` includes all the necessary components for training segmentation models from start to finish, including data augmentation transformations and transfer learning. Available architectures include: 2D U-Net [@Ronneberger2015unet], 3D U-Net [@isensee2017brain], ResNet [@he2016resnet], DenseNet [@Huang2017densenet], Count-ception [@Cohen2017countception], and HeMIS U-Net. These models can easily be enriched via attention blocks [@oktay2018attention] or FiLM layers (which modulate U-Net features using metadata). To facilitate the training process, `ivadomed` offers multiple loss functions such as the Dice coefficient [@milletari2016v], cross-entropy, and L2 norm, including some adapted to medical imaging challenges, such as the adaptive wing loss [@wang_adaptive_2019] for soft labels and the focal Dice loss [@wong20183d] for class imbalance. To partly address the problem of small datasets, mixup [@zhang2017mixup] has been modified to handle segmentation tasks. To mitigate class imbalance, `ivadomed` supports cascaded architectures. With a single inference, it is possible to narrow down the region of interest via object detection, and then segment a specific structure, as shown in Figure 2. In case of interruption during training, all parameters are saved after each epoch so that training can be resumed at any time. 
```

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+`ivadomed` includes all the necessary components for training segmentation models from start to finish. The first step usually consists of applying data augmentation transformations, such as affine, elastic, and ground-truth dilation, which are all included in `ivadomed`. The next step is model training, which can be done either from scratch or using transfer learning on a pre-trained model by fr ...
+
+It can be overwhelming to get started and choose across all the available models, losses, and parameters. `ivadomed`'s repository includes the script `ivadomed_automate_training` to configure and launch multiple trainings across GPUs. In addition, `ivadomed` includes tutorials that cover typical training scenarios such as one-class segmentation, cascade of architectures, and uncertainty estimatio ...
```
If we need space i think it can be removed
```suggestion
```

charleygros replied:
`ivadomed_automate_training`: This is quite a cool feature that we have not seen in other repo'. But I agree to remove the tutorial sentence!

jcohenadad commented on docs/source/index.rst:
```diff
@@ -29,6 +29,7 @@ Home
 
    purpose.rst
    technical_features.rst
+   use_cases.rst
```
👍  for putting it here

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,91 @@
+
+# Usage
+
+Past or ongoing research projects using `ivadomed` are listed [here](https://github.com/ivadomed/ivadomed/docs/source/use_cases.rst).
```
will need to be replaced by the built html URL on RTD

charleygros committed:
Update paper.md

Co-authored-by: Julien Cohen-Adad <jcohen@polymtl.ca>

charleygros committed:
Update paper.md

Co-authored-by: Julien Cohen-Adad <jcohen@polymtl.ca>

charleygros committed:
Update paper.md

Co-authored-by: Julien Cohen-Adad <jcohen@polymtl.ca>

charleygros committed:
Update paper.md

Co-authored-by: Julien Cohen-Adad <jcohen@polymtl.ca>

charleygros committed:
paper.md: per feature(s)

charleygros committed:
paper.md: add Andy fig

charleygros committed:
paper.md: fig autoref

charleygros committed:
paper.md: rm tutorial sentence

charleygros committed:
paper.bib: rm unused ref

charleygros commented:
~1130 words in [d678c76](https://github.com/ivadomed/ivadomed/pull/502/commits/d678c766074a0ab20fe5b8f206ebd804848b27ad).

jcohenadad committed:
Moved lemay_2020.png to dedicated repository for images

From now on, in order to keep the main repos size small, I suggest we move images/binaries to this repos:
https://github.com/ivadomed/doc-figures

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+Another challenge of medical imaging is the heterogeneity of the data across clinical centers, in terms of image features (e.g., contrast, resolution) and population demographics. This makes it challenging to create models that can generalize well across the many existing datasets. Recent cutting-edge methods address this problem, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], howeve ...
+
+![Overview of `ivadomed` main features.\label{fig:overview}](https://github.com/ivadomed/ivadomed/raw/master/images/overview.png)
```
update URL, see https://github.com/ivadomed/ivadomed/issues/511

jcohenadad replied:
done in b09a6e495a8f294050cc27ae41b2735f55b22a18

jcohenadad committed:
Updated URL for lemay_2020.png

jcohenadad committed:
Updated URL for overview.png, moved binary to other repo

jcohenadad committed:
Included comparison with other toolboxes

jcohenadad committed:
Removed repetition from abstract

jcohenadad committed:
Removed repetition

jcohenadad committed:
removed technical details

jcohenadad committed:
Refactored Evaluation section

jcohenadad committed:
Reduced length

jcohenadad committed:
Reduced length in statement of need

jcohenadad committed:
Removed technical details

jcohenadad committed:
Removed repetition in Loader section

jcohenadad committed:
Removed meat in Evaluation

jcohenadad reviewed (approved):
I aggressively edited, now arriving at 1037 words, which I think is acceptable. @charleygros please make sure to review my changes in case I was too aggressive at some places 😬

charleygros commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+# Statement of need
+
+Deep learning applied to medical imaging presents many challenges: datasets are often not publicly-available, ground truth labels are difficult to produce, and needs are usually tailored to particular datasets and tasks [@kim_deep_2019]. There already exists a few deep learning frameworks for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison ...
```
```suggestion
Deep learning applied to medical imaging presents many challenges: datasets are often not publicly-available, ground truth labels are difficult to produce, and needs are usually tailored to particular datasets and tasks [@kim_deep_2019]. There already exists a few deep learning frameworks for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison-with-other-projects). `ivadomed` notably address unmet needs in terms of data management, readily-available uncertainty outputs, missing modalities (in case of multi-channel training) and model comparison, to only name few of the original features.
```

charleygros commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+Deep learning applied to medical imaging presents many challenges: datasets are often not publicly-available, ground truth labels are difficult to produce, and needs are usually tailored to particular datasets and tasks [@kim_deep_2019]. There already exists a few deep learning frameworks for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison ...
+
+Another challenge of medical imaging is the heterogeneity of the data across hospitals (contrast, resolution, etc.), making it difficult to create models that can generalize well. Recent cutting-edge methods address this problem, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], however they are usually not implemented within a comprehensive framework. In addition to providing these arc ...
```
```suggestion
Another challenge of medical imaging is the heterogeneity of the data across hospitals (e.g., contrast, resolution), making it difficult to create models that can generalize well. Recent cutting-edge methods address this problem, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], however they are usually not implemented within a comprehensive framework. In addition to providing these architectures, `ivadomed` features losses adapted to imbalanced classes and non-binary predictions.
```

jcohenadad committed:
Update paper.md

Co-authored-by: charleygros <charley.gros@gmail.com>

jcohenadad committed:
Update paper.md

Co-authored-by: charleygros <charley.gros@gmail.com>

charleygros commented:
> I aggressively edited, now arriving at 1037 words, which I think is acceptable. @charleygros please make sure to review my changes in case I was too aggressive at some places grimacing

Thanks for the aggressive edition. I reviewed it: looks like it still stands up proudly and consistently. All good for me!

charleygros committed:
Merge branch 'master' of https://github.com/neuropoly/ivado-medical-imaging into all/joss_submission

charleygros commented:
FYI: the[ review process](https://joss.readthedocs.io/en/latest/submitting.html#the-review-process).

charleygros commented:
JOSS checklist (i.e., what we will be evaluated for): [here](https://joss.readthedocs.io/en/latest/review_checklist.html#software-paper).

I went through it and.. I feel quite good about it :-)

charleygros commented:
Note: you can visualize the compiled manuscript using this [beautiful tool](https://whedon.theoj.org/).

olix86 committed:
add orcid olivier

andreanne-lemay committed:
al: add orcid

andreanne-lemay commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+**Standard data structure:** In machine learning, lots of time is spent curating the data (renaming, filtering per feature) before entering the training pipeline [@Willemink2020-au]. `ivadomed` features an advanced data loader compatible with a standardized data structure in neuroimaging: the Brain Imaging Data Structure (BIDS) [@bids_2016]. Thus, any dataset following the BIDS convention can rea ...
+
+**Access to metadata:** One benefit of the BIDS convention is that each image file is associated with a JSON file containing metadata. `ivadomed`'s loader can parse image metadata (e.g., acquisition parameters, image contrast, resolution) and subject metadata (e.g., pathology, age, sex) for custom data splitting or extra information during training and evaluation. It is possible to modulate speci ...
```
```suggestion
**Access to metadata:** One benefit of the BIDS convention is that each image file is associated with a JSON file containing metadata. `ivadomed`'s loader can parse image metadata (e.g., acquisition parameters, image contrast, resolution) and subject metadata (e.g., pathology, age, sex) for custom data splitting or extra information during training and evaluation. It is possible to modulate specific layers of a convolutional neural network using metadata information to tailor it towards a particular data domain or to enable experiments with architectures such as FiLM [@perez2017film]. Metadata could also be useful to mitigate class imbalance via data balancing techniques.
```
Pretty implicit + it is mention 3 or 4 times in the article that ivadomed has FiLM

andreanne-lemay commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+**Architectures:** `ivadomed` includes all the necessary components for training segmentation models from start to finish, including data augmentation transformations and transfer learning. Available architectures include: 2D U-Net [@Ronneberger2015unet], 3D U-Net [@isensee2017brain], ResNet [@he2016resnet], DenseNet [@Huang2017densenet], Count-ception [@Cohen2017countception], and HeMIS U-Net. T ...
+
+**Losses and class imbalance:** Popular losses are available: Dice coefficient [@milletari2016v], cross-entropy, and L2 norm, including some adapted to medical imaging challenges, such as the adaptive wing loss [@wang_adaptive_2019] for soft labels and the focal Dice loss [@wong20183d] for class imbalance. Useful with small datasets, mixup [@zhang2017mixup] was modified to handle segmentation tas ...
```
```suggestion
**Losses and class imbalance:** Popular losses are available: Dice coefficient [@milletari2016v], cross-entropy, and L2 norm, including some adapted to medical imaging challenges, such as the adaptive wing loss [@wang_adaptive_2019] for soft labels and the focal Dice loss [@wong20183d] for class imbalance. Useful to alleviate overfitting, mixup [@zhang2017mixup] was modified to handle segmentation tasks. To mitigate class imbalance, `ivadomed` supports cascaded architectures. With a single inference, it is possible to narrow down the region of interest via object detection and then segment a specific structure, as illustrated in \autoref{fig:lemay2020}.
```
Why is it useful on small datasets?

andreanne-lemay commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+# Statement of need
+
+Deep learning applied to medical imaging presents many challenges: datasets are often not publicly-available, ground truth labels are difficult to produce, and needs are usually tailored to particular datasets and tasks [@kim_deep_2019]. There already exists a few deep learning frameworks for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison ...
```
```suggestion
Deep learning applied to medical imaging presents many challenges: datasets are often not publicly-available, ground truth labels are difficult to produce, and needs are usually tailored to particular datasets and tasks [@kim_deep_2019]. There already exists a few deep learning frameworks for medical imaging, but [each has their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison-with-other-projects). `ivadomed` notably addresses unmet needs in terms of data management, readily-available uncertainty outputs, missing modalities (in case of multi-channel training) and model comparison, to only name few of the original features.
```

andreanne-lemay commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+**Uncertainty:** Aleatoric [@wang_aleatoric_2019] and/or epistemic [@nair_exploring_2018] uncertainties can be computed voxel-wise and/or object-wise [@roy_quicknat_2018]. Multiple metrics are available, including entropy and coefficient of variation.
+
+**Post-processing:** Predictions can be conveniently filtered using popular methods, e.g. fill holes, remove small objects, threshold using uncertainty. It is also possible to compute metrics for specific object sizes (e.g., small vs. large lesions). `ivadomed` has a module to find the optimal threshold value on the output soft prediction, via a grid-search finding applied to evaluation metrics o ...
```
```suggestion
**Post-processing:** Predictions can be conveniently filtered using popular methods, e.g., fill holes, remove small objects, threshold using uncertainty. It is also possible to compute metrics for specific object sizes (e.g., small vs. large lesions). `ivadomed` has a module to find the optimal threshold value on the soft output prediction, via a grid-search finding applied to evaluation metrics or ROC curve.
```
or simply `soft prediction`

andreanne-lemay reviewed (commented):
Nice job on reducing the word count! The essence of ivadomed is well presented and clear even with 300+ words less.

charleygros committed:
Update paper.md

Co-authored-by: andreanne-lemay <49137243+andreanne-lemay@users.noreply.github.com>

charleygros committed:
Update paper.md

Co-authored-by: andreanne-lemay <49137243+andreanne-lemay@users.noreply.github.com>

charleygros committed:
Update paper.md

Co-authored-by: andreanne-lemay <49137243+andreanne-lemay@users.noreply.github.com>

charleygros committed:
Update paper.md

Co-authored-by: andreanne-lemay <49137243+andreanne-lemay@users.noreply.github.com>

olix86 reviewed (approved):
Good job! looks good to me 🚀

lrouhier reviewed (approved):
Fantastic 👍

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+Another challenge of medical imaging is the heterogeneity of the data across hospitals (e.g., contrast, resolution), making it difficult to create models that can generalize well. Recent cutting-edge methods address this problem, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], however they are usually not implemented within a comprehensive framework. In addition to providing these arc ...
+
+![Overview of `ivadomed` main features.\label{fig:overview}](https://raw.githubusercontent.com/ivadomed/doc-figures/main/overview.png)
```
now that we are no more restricted by space (but by words), i suggest we put back ivadomed's logo in the overview figure.

jcohenadad replied:
done in https://github.com/ivadomed/ivadomed/pull/502/commits/fcb1d87d5ee784c8d39263c4929c37f3202e89c5

andreanne-lemay committed:
Merge branch 'master' into all/joss_submission

jcohenadad commented on paper.md:
```diff
@@ -0,0 +1,89 @@
+
+# Usage
+
+Past and ongoing research projects using `ivadomed` are listed [here](https://github.com/ivadomed/ivadomed/docs/source/use_cases.rst). The figure below illustrates a cascaded architecture for segmenting spinal tumors on MRI data [@lemay_fully_2020].
```
we need to redirect to RTD

jcohenadad replied:
```suggestion
Past and ongoing research projects using `ivadomed` are listed [here](https://ivadomed.org/en/stable/use_cases.html). The figure below illustrates a cascaded architecture for segmenting spinal tumors on MRI data [@lemay_fully_2020].
```

andreanne-lemay committed:
Update paper.md

Co-authored-by: Julien Cohen-Adad <jcohen@polymtl.ca>

jcohenadad committed:
paper.md: now linking overview figure with logo
